internal/output/markdowntable.go

```diff
@@ -12,12 +12,20 @@ import (
 func PrintMarkdownTableResults(vulnResult *models.VulnerabilityResults, outputWriter io.Writer) {
 	outputTable := table.NewWriter()
 	outputTable.SetOutputMirror(outputWriter)
-	outputTable.AppendHeader(table.Row{"OSV URL", "CVSS", "Ecosystem", "Package", "Version", "Source"})
-
 	outputTable = tableBuilder(outputTable, vulnResult, false)
 
-	if outputTable.Length() == 0 {
+	if outputTable.Length() != 0 {
+		outputTable.RenderMarkdown()
+	}
+
+	outputLicenseTable := table.NewWriter()
+	outputLicenseTable.SetOutputMirror(outputWriter)
+	outputLicenseTable.AppendHeader(table.Row{"License", "No. of Package Versions"})
```

Comment:
Why is this line needed here but not in table.go?

Reply:
that's a mistake, removed this line